ste/xfer-remoteToLocal-file.go

```diff
@@ -171,7 +171,7 @@ func remoteToLocal_file(jptm IJobPartTransferMgr, p pipeline.Pipeline, pacer pac
 		// to correct name.
 		pseudoId := common.NewPseudoChunkIDForWholeFile(info.Source)
 		jptm.LogChunkStatus(pseudoId, common.EWaitReason.CreateLocalFile())
-		dstFile, err = createDestinationFile(jptm, info.getTempDownloadPath(), fileSize, writeThrough)
+		dstFile, err = createDestinationFile(jptm, info.getDownloadPath(), fileSize, writeThrough)
```

Comment:
I liked the previous name better. It gives clear idea that we're talking about temp download directory and not the actual destination directory.

Reply:
The value of env var dictates if we're downloading to a temp or the actual path. We're not downloading to temp path in all cases.